### PR TITLE
Immutable classes should by definition be final, Fix #20

### LINKED ISSUE #20: Immutable classes should by definition be final

### PR DESCRIPTION
**Pull request by minborg**

### CODE REVIEW

File: src/main/java/net/openhft/chronicle/jlbh/ImmutableJLBHResult.java [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 import static java.util.Collections.unmodifiableMap;
 
-class ImmutableJLBHResult implements JLBHResult {
+final class ImmutableJLBHResult implements JLBHResult {
```
Comment (minborg) [inline src/main/java/net/openhft/chronicle/jlbh/ImmutableJLBHResult.java:28]: Classes are packet private so we know they are never subclassed externally

Comment (RobAustin) [reply src/main/java/net/openhft/chronicle/jlbh/ImmutableJLBHResult.java:28]: ugg !

Comment (RobAustin) [reply src/main/java/net/openhft/chronicle/jlbh/ImmutableJLBHResult.java:28]: you know what I think of final classes! :-)

**Review by RobAustin — APPROVED** [review]

File: src/main/java/net/openhft/chronicle/jlbh/ImmutableJLBHResult.java (hunk 1 above)
Comment (JerryShea) [reply src/main/java/net/openhft/chronicle/jlbh/ImmutableJLBHResult.java:28]: I agree with you Rob!

**Review by JerryShea — APPROVED** [review]
Begrudgingly approving ;)

File: src/main/java/net/openhft/chronicle/jlbh/ImmutableJLBHResult.java (hunk 1 above)
Comment (minborg) [reply src/main/java/net/openhft/chronicle/jlbh/ImmutableJLBHResult.java:28]: See the subject. If a class is extendable, then, by definition, instances of the class can exist, that is mutable.

**Pull request merged**